browser-sign-in/app/src/main/java/com/okta/android/samples/browser_sign_in/views/LoginFragment.kt

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2022-Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.okta.android.samples.browser_sign_in.views
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.navigation.fragment.findNavController
+import com.okta.android.samples.browser_sign_in.data.LoginState
+import com.okta.android.samples.browser_sign_in.data.LoginViewModel
+import com.okta.android.samples.browser_sign_in.databinding.FragmentLoginBinding
+import timber.log.Timber
+
+class LoginFragment : Fragment() {
+    private val viewModel: LoginViewModel by viewModels()
+    private lateinit var binding: FragmentLoginBinding
```

Comment:
We are likely leaking this binding/context. It should be cleared `onDestroyView`. See https://github.com/okta/okta-mobile-kotlin/blob/master/app/src/main/java/sample/okta/android/util/BaseFragment.kt